sitemedia/js/snippets/init_gtags.js

```diff
@@ -0,0 +1,8 @@
+/*
+  Google Analytics gtags snippet to be included if analytics are enabled.
+*/
+window.dataLayer = window.dataLayer || [];
+function gtag(){dataLayer.push(arguments);}
+gtag('js', new Date());
+
+gtag('config', 'UA-87887700-5');
```

Comment:
Since we're adding a local settings variable and setting it in the context anyway, what do you think of setting the UA code in the variable instead of hard-coding it in the javascript? Any benefit, or not something that should change?

Reply:
In other words something like `GA_TRACKING_ID` which when set passes that variable into the js and when `None` turns off analytics? Sound good to me!

Reply:
I thought about that and waffled. It probably shouldn't ever be changing outside of a release cycle, but I like the flexibility just in case. When I pick this back up, I'll refactor the js into the variable so we can change it if need be.

Reply:
Yes, if `None` or not set it would just turn off analytics.  I agree we don't really expect it to change, but I like having it in a config instead of code.